AWSAppSyncClient/AWSAppSyncClientConfigurationError.swift

```diff
@@ -22,7 +22,7 @@ extension AWSAppSyncClientConfigurationError: LocalizedError {
         switch self {
         case .invalidAuthConfiguration(let message):
             return "Invalid Auth Configuration: \(message)"
-        case .invalidClientDatabasePrefixConfiguration(let message):
+        case .clientDatabasePrefixAlreadyInUse(let message):
             return "Invalid Client Database Prefix Configuration: \(message)"
```

Comment:
message should reflect that the cache configuration is in use.

Reply:
changed.